lms/static/certificates/sass/_layouts.scss

```diff
@@ -0,0 +1,324 @@
+// ------------------------------
+// // Open edX Certificates: Layouts
+
+// About: styling for generic layouts and responsive support.
+
+// #BASE
+// #ACCOMPLISHMENT
+
+// ------------------------------
+// #BASE
+// ------------------------------
+%layout-wrapper {
+    margin-bottom: spacing-vertical(base);
+    padding: 0 (gutter()*2);
+
+    @include susy-breakpoint($bp-medium, $susy) {
+        padding: 0 gutter();
+    }
+}
+
+%layout {
+    @include container();
+}
+
+// app header
+.wrapper-header {
+    @extend %layout-wrapper;
+    padding-top: spacing-vertical(small);
+    padding-bottom: spacing-vertical(small);
+}
+
+.header-app {
+    @extend %layout;
+
+    .logo {
+        width: span(2);
+        margin: 0 auto;
+    }
+}
+
+.header-app-title {
+    margin: 0;
+    text-align: center;
+}
+
+// banner
+.wrapper-banner {
+    @extend %layout-wrapper;
+    padding-top: spacing-vertical(small);
+    padding-bottom: spacing-vertical(small);
+
+    .banner {
+        @extend %layout;
+    }
+}
+
+// message to user-centric banner
+.wrapper-banner-user {
+    margin-top: -(spacing-vertical(base)); // abutt the global header, plz
```

Comment:
NIT: Too much "T" on that abutment. You can get away with one less T, Mr. http://bit.ly/1PdZ8E0